code/game/objects/items/robot/robot_parts.dm

```diff
@@ -226,6 +226,8 @@
 
 			if(O.mind && O.mind.special_role)
 				O.mind.store_memory("As a cyborg, any objectives listed here are null and void, and will be marked as failed. They are simply here for memory purposes.")

```

Comment:
Technically removing this line is more proper than adding the others, as borgs antags are still antags in every code sense of the word.

A line saying that laws come before objectives isn't a bad idea though.